subsys/bluetooth/host/hci_core.c

```diff
@@ -85,6 +85,7 @@ static bt_le_scan_cb_t *scan_dev_found_cb;
 static u8_t pub_key[64];
 static struct bt_pub_key_cb *pub_key_cb;
 static bt_dh_key_cb_t dh_key_cb;
+static u8_t dd_ch_map;
```

Comment:
I think this would better go into struct bt_dev since. What does "dd" stand for?

Reply:
device discovery, I can change it to adv though, since the scanner/initiator has no API for setting channel map.